extension/js/extension.js

```diff
@@ -68,6 +68,10 @@ function filter_kernel(project) {
                 if (env != name) {
                     $('#' + idList[i]).css("display", "none");
                 }
+                else {
+                    $('#' + idList[i]).find('a').text('Python 3');
```

Comment:
You change the text. It's fine, as the user should only see the kernel name.

But I have questions:
* If I'm in CERNBox tab I will see all kernels, right? But if I move in a Project, get out, enter another one and get out, then the list will be filled with `Python 3` kernels in the list, no?
* I can have other kernels besides Py3 in the conda env, right? Won't this change the name to Py3? Couldn't you just remove the `[_conda_env_uid_]` text?

Reply:
I shall be creating a new PR with these changes soon.